src/bioregistry/data/bioregistry.json

```
@@ -14623,6 +14623,38 @@
       "uri_format": "https://www.cellrepo.com/repositories/$1"
     }
   },
+  "cellxgene": {
+    "contributor": {
+      "email": "robert.j.allaway@gmail.com",
+      "github": "allaway",
+      "name": "Robert Allaway",
+      "orcid": "0000-0003-3573-3565"
+    },
+    "description": "CELLxGENE is an interactive data visualization and exploration tool developed by the Chan Zuckerberg Initiative that enables researchers to analyze and share single-cell genomics datasets. It provides a user-friendly interface for biologists and computational scientists to interrogate gene expression patterns across different cell types. Identifiers correspond to collections made available in CELLxGENE.",
+    "example": "436154da-bcf1-4130-9c8b-120ff9a888f2",
+    "github_request_issue": 1541,
+    "homepage": "https://cellxgene.cziscience.com",
```

this page is more specifically about the collections semantic space

```suggestion
    "homepage": "https://cellxgene.cziscience.com/collections",
```